web/components/SchedulePage.jsx

```diff
@@ -1,19 +1,22 @@
 import React from 'react';
 import moment from 'moment';
 import { connect } from 'react-redux';
+import { Check } from 'styled-icons/feather/Check.cjs'
 
 import ScheduleHeader from './ScheduleHeader';
 import SideBar from './SideBar';
 import Flex from './Flex';
 import SetBlock from './SetBlock';
 import Text from './Text';
-
-import { fetchCurrentTeamMemberById, setSelectedDay } from '../reducers/environment';
 import LoadingDots from './Loading';
 
+import { fetchCurrentTeamMemberById, setEditModeSchedule, setSelectedDay } from '../reducers/environment';
+import Card from './Card';
+
 class SchedulePage extends React.Component {
     state = {
         daysOfWeek: [],
+        setBlocksForEdit: {}
```

Comment:
Good idea to make this into a new array, but remember that Redux is there to help organize state and there may be child components that want to know if there is anything in this piece of state. 

So let's move this into Redux which also helps separation of concerns by keeping our state management and our UI in separate files. Also a better name would be `setBlocksToEdit` or maybe `setBlocksToUpdate` or `unsavedSetblocks`